static/app/views/replays/replaySearchBar.tsx

```diff
@@ -123,6 +124,12 @@ function ReplaySearchBar(props: Props) {
       maxMenuHeight={500}
       hasRecentSearches
       fieldDefinitionGetter={getReplayFieldDefinition}
+      mergeSearchGroupWith={{
+        click: {
+          documentation: t('Requires min sdk >= 7.44.0'),
```

Comment:
```suggestion
          documentation: t('Requires SDK version >= 7.44.0'),
```

Reply:
Wonder if we should just list our current latest SDK...

```suggestion
          documentation: t('Requires SDK version >= 7.48.0'),
```

Reply:
I've changed the wording, but gonna leave it aligned with the other 7.44.0 messaging we have so we're consistent.